Source/WebCore/css/parser/CSSParserContext.cpp

```diff
@@ -109,6 +109,9 @@ CSSParserContext::CSSParserContext(const Document& document, const URL& sheetBas
     , grammarAndSpellingPseudoElementsEnabled { document.settings().grammarAndSpellingPseudoElementsEnabled() }
     , customStateSetEnabled { document.settings().customStateSetEnabled() }
     , thumbAndTrackPseudoElementsEnabled { document.settings().thumbAndTrackPseudoElementsEnabled() }
+#if ENABLE(SERVICE_CONTROLS)
```

Comment:
Please use `#if ENABLE(SERVICE_CONTROLS)` consistently everywhere `imageControlsEnabled` is being set/read.